package.json

```diff
@@ -411,6 +411,12 @@
         "title": "Configure the compilation database for C and C++ analysis",
         "category": "SonarQube"
       },
+      {
+        "command": "SonarLint.RefreshAIAgentsConfiguration",
```

Comment:
It is too complicated to listen to both MCP config file changes & the rules file changes across the different IDEs, etc. So I went for an explicit refresh button.